homeassistant/components/intellifire/sensor.py

```diff
@@ -126,3 +94,40 @@ async def async_setup_entry(
         value_fn=_time_remaining_to_timestamp,
     ),
 )
+
+
+class IntellifireSensor(
+    IntellifireEntity, SensorEntity, IntellifireSensorRequiredKeysMixin
+):
+    """Extends IntellifireEntity with Sensor specific logic."""
+
```

Comment:
If you add the type hint back in like this, you shouldn't need self.description anymore. You should even be able to drop the `__init__` method.
```suggestion
class IntellifireSensor(
    IntellifireEntity, SensorEntity, IntellifireSensorRequiredKeysMixin
):
    """Extends IntellifireEntity with Sensor specific logic."""

    entity_description: IntellifireSensorEntityDescription

```